src/main/java/pilcrow/Pilcrow.java

```diff
@@ -0,0 +1,91 @@
+package pilcrow;
+
+import java.util.Scanner;
+
+/**
+ * Pilcrow is a personal assistant chatbot made for CS2103, based off of Duke.
+ */
+public class Pilcrow {
+    private Ui ui;
+    private Storage storage;
+    private TaskList taskList;
+
+    private final static String PILCROW_FILE_PATH = "data/pilcrow.txt";
+
+    private Pilcrow() {
+        this.ui = new Ui();
+        this.storage = new Storage(Pilcrow.PILCROW_FILE_PATH);
+        this.taskList = new TaskList(this.storage.load());
+    }
+
+
+    public static void main(String[] args) {
+        Pilcrow pilcrow = new Pilcrow();
+        pilcrow.run();
+    }
+
+    /**
+     * Runs the main body of the Pilcrow script.
+     */
+    private void run() {
+        Boolean isExit = false;
+        Scanner scanner = new Scanner(System.in);
+
+        while (!isExit) {
+            try {
+                Parser parser = new Parser(scanner.nextLine());
+                Pilcrow.runCommand(parser.getCommandWord(), parser.getRestOfCommand(),
+                        this.ui, this.storage, this.taskList, parser);
+
+                isExit = (parser.getCommandWord().equals("bye"));
+                if (isExit) {
+                }
```

Comment:
Is there supposed to be some logic in this `if` block?

Reply:
You should remove this block if there is no use for it.

Reply:
Good idea, I'll do that.